cwms_radar_api/src/test/java/fixtures/RadarApiSetupCallback.java

```diff
@@ -35,11 +39,11 @@ public class RadarApiSetupCallback implements BeforeAllCallback,AfterAllCallback
 
     private static TomcatServer radarInstance;
     private static CwmsDatabaseContainer cwmsDb;
-    private static TestRealm realm;
-    private static TestAuthValve authValve;
-    private static final String ORACLE_IMAGE = System.getProperty("RADAR.oracle.database.image", CwmsDatabaseContainer.ORACLE_18XE);
+
+    private static final String ORACLE_IMAGE = System.getProperty("RADAR.oracle.database.image", CwmsDatabaseContainer.ORACLE_19C);
     private static final String ORACLE_VOLUME = System.getProperty("RADAR.oracle.database.volume", "cwmsdb_radar_volume");
-    private static final String CWMS_DB_IMAGE = System.getProperty("RADAR.cwms.database.image", "registry.hecdev.net/cwms_schema_installer:21.1.1");
+    private static final String CWMS_DB_IMAGE = System.getProperty("RADAR.cwms.database.image", "registry.hecdev.net/cwms/schema_installer:latest-dev");
+
```

Comment:
Does this mean we're always running against the latest version of the installer or the latest version of the schema or both?  I think I'd rather depend on a specific version than latest.

Reply:
That does mean the latest schema installer (basically what's in master). The tests currently do depend on it. But we should change it once a release that includes the changes is made.